modules/mono/editor/GodotTools/GodotTools/GodotSharpEditor.cs

```diff
@@ -337,6 +337,28 @@ public Error OpenInExternalEditor(Script script, int line, int col)
 
                             args.Add("--args");
                         }
+
+                        // Try VSCodium as a fallback if Visual Studio Code can't be found.
+                        if (!macOSAppBundleInstalled)
+                        {
+                            const string vscodiumBundleId = "com.vscodium.codium";
```

Comment:
Constants should be `PascalCase` (there might still be `camelCase` ones in the codebase, it was half-and-half, and we've properly picked one recently).
```suggestion
                            const string VscodiumBundleId = "com.vscodium.codium";
```